api-tests/tools/cmake/compiler/GNUARM_LINUX.cmake

```diff
@@ -0,0 +1,54 @@
+#/** @file
+# * Copyright (c) 2020, Arm Limited or its affiliates. All rights reserved.
+# * SPDX-License-Identifier : Apache-2.0
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *  http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+#**/
+
+set(CMAKE_SYSTEM_NAME "LINUX")
+set(CMAKE_SYSTEM_PROCESSOR "ARM")
+
+if (NOT DEFINED GCC_TOOLCHAIN_NAME)
```

Comment:
PSA_NS_OS=LINUX and TOOLCHAIN_NAME=arm-poky-linux-musleabi-gcc
please set it in this file, instead of cmake command line